packages/applications-service-api/prisma/schema.prisma

```diff
@@ -61,40 +61,13 @@ model Representation {
   referenceId                    String?
   status                         String?
   dateReceived                   DateTime?
-  originalRepresentation         String    @db.NVarChar(4000)
-  redacted                       Boolean   @default(false)
-  redactedRepresentation         String?   @db.NVarChar(4000)
-  redactedBy                     String?
-  redactedNotes                  String?   @db.NVarChar(4000)
+  representationComment          String?   @db.NText
```

Comment:
`NText` is deprecated which is why it was `NVarChar(4000)` before. Back Office are using `NText` so we can go for consistency (and both have to switch when it is fully deprecated), or use `NVarChar` which is the [recommended replacement](https://learn.microsoft.com/en-us/sql/t-sql/data-types/ntext-text-and-image-transact-sql?view=sql-server-ver16) and preempt it

Reply:
Originally my concern was that BO could have a 4100 char rep and we would truncate.  Thought I'd make it consistent now then we can align on length in future.